src/integration/java/io/pinecone/integration/controlPlane/pod/CollectionTest.java

```diff
@@ -129,18 +124,16 @@ public void testIndexFromDifferentMetricCollection() throws InterruptedException
         IndexMetric[] metrics = {IndexMetric.COSINE, IndexMetric.EUCLIDEAN, IndexMetric.DOTPRODUCT};
         IndexMetric targetMetric = IndexMetric.COSINE;
         for (IndexMetric metric : metrics) {
```

Comment:
This feels kind of pointless now since we're just declaring the metric here in the test class. Can probably just pass a different metric explicitly rather than having this weird logic that I was using at one point. 😅

Reply:
True! I'll update

Reply:
Updated